### ISSUE TITLE
Unable to set number of days in preferences, Firefox 85.0

### ISSUE THREAD

**Issue by jsalomons** [Rayquaza01/HistoryCleaner#11]
No option to set days, see attached image.
![Capture](https://user-images.githubusercontent.com/44492674/107762314-b9e56180-6d24-11eb-838d-5a0707d90ed5.PNG)

**Comment by Rayquaza01** [comment]
That is strange.
What Firefox version and OS are you using?
Does this still happen after restarting the browser? How about if you uninstall and reinstall the extension?

**Comment by jsalomons** [comment]
Windows 10, Firefox version 85.0.1 64 bit.

Not sure what's happening - I uninstalled and re-installed about 4 or 5 times. Reason I tried doing it multiple times was because I caught a glimpse of the options the first time I did it, only for them to disappear once I clicked on the 'Allow this extension to run in private windows' checkbox and then clicked 'Okay, Got It'. The last re-install and they were there.

It occurred to me that other extensions could be interfering but think it's considered a privileged page in Firefox so none of them appear to be running. Hopefully no one else experiences it.

Thanks for getting back to me so quickly and really appreciate you taking the time to create this extension :)
